### PR TITLE
[CTOR-886]- refresh SAP HANA doc page

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: pp/integrations/plugin-packs/procedures/applications-databases-sap-hana.md [hunk 1]
```diff
@@ -2,18 +2,127 @@
...
+* **App-DB-Sap-Hana-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="App-DB-Sap-Hana-custom" label="App-DB-Sap-Hana-custom">
+
+| Service Alias        | Service Template                            | Service Description                     |
+|:---------------------|:--------------------------------------------|:----------------------------------------|
+| Blocked-Transactions | App-DB-Sap-Hana-Blocked-Transactions-custom | Check total  blocked transactions       |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-sap-hana.md:23]:
```suggestion
| Blocked-Transactions | App-DB-Sap-Hana-Blocked-Transactions-custom | Check total blocked transactions       |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-sap-hana.md [hunk 2]
```diff
@@ -2,18 +2,127 @@
...
+| Connected-Users      | App-DB-Sap-Hana-Connected-Users-custom      | Check total connected users             |
+| Connection-Time      | App-DB-Sap-Hana-Connection-Time-custom      | Check the connection time to the server |
+| Disk-Usage           | App-DB-Sap-Hana-Disk-Usage-custom           | Check disk usage                        |
+| Host-Cpu             | App-DB-Sap-Hana-Host-Cpu-custom             | Check host cpu usage                    |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-sap-hana.md:27]:
```suggestion
| Host-Cpu             | App-DB-Sap-Hana-Host-Cpu-custom             | Check host CPU usage                    |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-sap-hana.md [hunk 3]
```diff
@@ -2,18 +2,127 @@
...
+| *volume*#volume.usage.bytes | B     |
+
+> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.
+
+</TabItem>
+</Tabs>
+
+## Prerequisites
 
 To monitor Sap Hana DB, you need to install SAP Hana Linux Client. The client is
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-sap-hana.md:127]:
```suggestion
To monitor SAP Hana DB, you need to install the SAP Hana Linux Client. The client is available
```

File: pp/integrations/plugin-packs/procedures/applications-databases-sap-hana.md [hunk 4]
```diff
@@ -2,18 +2,127 @@
...
 on SAP support website (an account is needed).
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-sap-hana.md:128]:
```suggestion
on the SAP support website (an account is needed).
```

Comment (cg-tw) [reply pp/integrations/plugin-packs/procedures/applications-databases-sap-hana.md:128]: same as FR, what is the command just below?

File: pp/integrations/plugin-packs/procedures/applications-databases-sap-hana.md [hunk 5]
```diff
@@ -24,35 +133,440 @@ yum install unixODBC perl-DBD-ODBC
 
 ### Configuration of freetds.conf file
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-sap-hana.md:134]: freetds.conf or odbcinst.ini?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/applications-databases-sap-hana.md [hunk 6]
```diff
@@ -2,57 +2,573 @@
...
 on SAP support website (an account is needed).
 
+Run the following command to install the necessary tools to connect to the SAP HANA database via ODBC, as well as the Perl module to enable the use of Perl scripts with this connection.
+
 ``` shell
 yum install unixODBC perl-DBD-ODBC
 ```
 
-### Configuration of freetds.conf file
+### Configuration of odbcinst.ini file
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/applications-databases-sap-hana.md:136]:
```suggestion
### Configuration file
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**